frontend/src/Frontend/Store/V1.hs

```diff
@@ -0,0 +1,176 @@
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE TemplateHaskell #-}
+module Frontend.Store.V1 where
+
+import Data.Aeson
+import Data.Aeson.GADT.TH
+import qualified Data.ByteString.Base16 as Base16
+import Data.Constraint (Dict(Dict))
+import Data.Constraint.Extras
+import Data.Dependent.Map (DMap, DSum(..))
+import qualified Data.Dependent.Map as DMap
+import Data.Functor.Identity (Identity(Identity), runIdentity)
+import qualified Data.IntMap as IntMap
+import Data.Map (Map)
+import qualified Data.Map as Map
+import Data.Maybe (catMaybes)
+import Data.Text (Text)
+import qualified Data.Text as T
+import qualified Data.Text.Encoding as T
+
+import Common.Wallet
+import Common.Network (NetworkName, NodeRef)
+import Common.OAuth (OAuthProvider(..))
+import Common.GistStore (GistMeta)
+
+import Frontend.Store.TH
+import qualified Frontend.Store.V0 as V0
+import qualified Frontend.Store.V0.Wallet as V0
+import Frontend.Store.MigrationUtils
+
+-- WARNING: Upstream deps. Check this when we bump pact and obelisk!
```

Comment:
Tests will catch breakage from upstream bumps right?

Reply:
They should do, yeah.